testing/README.md

```diff
@@ -0,0 +1,95 @@
+# How to Test E2E Resource Changes
+This guide will give a step by step instruction on how to test changes made to E2E testing resources before pushing a PR. 
+
```

Comment:
Could be useful to add a message/list of what this guide will entail, or even a table of contents, e.g. "To test changes to the E2E tests, you will need to fork the repo, set up IAM roles and an EKS cluster, and build sample apps into ECRs/S3 buckets" and anything else of relevance.

Reply:
Done